source/code-snippets/usage-examples/deleteMany.ts

```diff
@@ -1,22 +1,34 @@
+/* Delete multiple documents */
+
+// Import the MongoClient type from the mongodb package.
 import { MongoClient } from "mongodb";
 
 // Replace the uri string with your MongoDB deployment's connection string.
 const uri = "<connection string uri>";
 
+// Create a new client and connect to MongoDB.
 const client = new MongoClient(uri);
 
+// Define the Movie interface to model the data.
```

Comment:
Suggestion:

I'm open to any discussion, but I think some of the information is self-apparent. I think "Create a sample interface" contains enough information.


Also, this code isn't used in this example, so I think it could be good to remove (feel free to ticket if it's not trivial to fix and check).